ElasticaBundle/Resources/config/populator.yml

```diff
@@ -11,6 +11,5 @@ services:
         arguments:
             - @open_orchestra_elastica.indexor.content
             - @open_orchestra_model.repository.content
-            - %open_orchestra_backoffice.orchestra_choice.front_language%
```

Comment:
I think that the content populator should be placed in the ElasticaAdminBundle, which is only loaded in the admin part of the website